ui/src/components/TestRunner/index.tsx

```diff
@@ -65,7 +59,7 @@ export const TestRunner = () => {
   const [supportedAssets, setSupportedAssets] = useState([] as string[]);
   const [supportedSeps, setSupportedSeps] = useState([] as number[]);
 
-  function resetAllState() {
+  const resetAllState = () => {
```

Comment:
using an arrow function for consistency and also bc arrow functions are best practice for preventing issues with scope and implicitly returning. This also allow TS to infer the return type rather than having to explicitly list it